tendrl/commons/flows/import_cluster/__init__.py

```diff
@@ -109,6 +112,46 @@ def run(self):
                 _cluster.current_job['status'] = "finished"
                 _cluster.is_managed = "yes"
                 _cluster.save()
+
+                # Creating a Job to do the profiling task. This job is
+                #  separate from the import flow.
+                if _cluster.volume_profiling_flag != "leave-as-is":
+                    if (_cluster.volume_profiling_state !=
+                            _cluster.volume_profiling_flag+"d"):
```

Comment:
We should add some comment here explaining why we are adding `d` at the last to check `_cluster.volume_profiling_state`. I feel its because valid values of `_cluster.volume_profiling_state` are `Enabled`, `Disabled` and `Mixed`.
Please add comment to explain the same.